docs/adr/005-use-django-for-back-end-web-framework.md

```diff
@@ -0,0 +1,45 @@
+# 5. Back-end web framework
+
+Date: 2021-07-02
+
+## Status
+
+Proposed
+
+## Context
+
+We discussed different options for our tech stack on GitHub as a team — see discussion in [#25](https://github.com/18F/OPRE-Unicorn/issues/25). Afterwards, the engineering team met to discuss back-end tech stack options in particular. 
+
+### Explanation of the tech concept
+
+Django, Flask, Rails, Sinatra -- these are all examples of web frameworks. Web frameworks are pre-written code that give developers tools and features right away when they start new web application projects. 
+
+Using a web framework, we won't be starting off writing code on a totally blank page. We'll start with some of the tools and features we need already in place, such as code to fetch and send data to and from a database. That will help us jump more quickly to writing project-specific code. 
+
+### Resources for further reading
+
++ https://www.djangoproject.com/
++ https://docs.djangoproject.com/en/dev/faq/general/#faq-general 
++ Django Admin: https://developer.mozilla.org/en-US/docs/Learn/Server-side/Django/Admin_site
+
+## Decision
+
+We found concensus on using Django, although we considered Flask as well. We felt that the features Django offers out-of-the-box -- such as tools to interact with the database layer, and the Django Admin dashboard -- are ones the project is likely to use and benefit from. 
```

Comment:
In the comments thread, Carly had left this comment, and wondering what came of it, esp given the uncertainty we still have around whether we will be going down the single page web app route.

> I've used Flask and have found it to be very easy to use with lots of reliable and readable documentation. It does take a bit more work than Django to get a Flask app connected to a database, but it's doable no doubt. It's very lightweight and would suggest this if we go the single page web app route.